src/pybind/mgr/rook/module.py

```diff
@@ -248,19 +248,16 @@ def get_hosts(self):
         return [orchestrator.InventoryNode(n, inventory.Devices([])) for n in self.rook_cluster.get_node_names()]
 
     @deferred_read
-    def describe_service(self, service_type=None, service_id=None, node_name=None, refresh=False):
+    def list_daemons(self, daemon_type=None, daemon_id=None, node_name=None, refresh=False):
```

Comment:
The change of name in this method to `list_daemon` seems to be wrong .. in the `orchestrator_cli` we are calling "describe_service"

Reply:
Both methods exist. ``list_daemons`` should describe individual daemons or pods, while ``describe_service`` should describe collections of daemons (e.g., CRDs, ReplicaSets, whatever).  This method is more like list_daemons that describe_service.

Reply:
thx. Understood